Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -326,6 +319,25 @@ bool ParsePartitionData(SPartition& _rPartition)
 	return ParsedOK;
 }
 
+void ParseFileSystemData(u64 offset, const IFileInfo& directory)
+{
+	for (const DiscIO::IFileInfo& file_info : directory)
+	{
+		const std::string path = file_info.GetPath();
+		DEBUG_LOG(DISCIO, "%s", path.empty() ? "/" : path.c_str());
```

Comment:
I know this is a debug log and you're just porting it, but it looks 1. very spammy; 2. very non-obvious in its meaning since it will just print a path. Can you either remove it or modify the log message to make it more explicit what's being logged?